src/Data/MIME.hs

```diff
@@ -78,6 +78,7 @@ module Data.MIME
   , headerCC
   , headerBCC
   , headerDate
+  , replyHeaderReferences
```

Comment:
Any suggestion for a better name I'm all ears.

Reply:
It's fine. The haddock explains it adequately.